sys-apps/grep/grep-3.7.ebuild

```diff
@@ -45,6 +45,11 @@ src_prepare() {
 		src/egrep.sh || die #523898
 
 	default
+
+	# touch generated files after patching m4, to avoid activating maintainer
+	# mode
+	touch ./aclocal.m4 ./config.hin ./configure || die
```

Comment:
config.hin? Check if it's not a typo, but if it's not, fine

Reply:
Yeah this is actual file in the source, it apparently means `config.h.in` :joy: